spec/commands/contribution_ranker/compute_score_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe ContributionRanker::ComputeScore do
+  def perform(*args)
+    described_class.for(*args)
+  end
+
+  context 'when user contributes nothing' do
+    it 'returns 0 when there are no other users' do
+      expect(perform(self_reviewed_prs: 30, per_user_reviews: [])).to eq(0)
+    end
+    it 'returns 0 when there are multiple other users' do
+      expect(perform(self_reviewed_prs: 30, per_user_reviews: [0, 0, 0])).to eq(0)
+    end
+  end
+
+  context 'when user contributes something' do
+    it 'is not 0' do
+      expect(perform(self_reviewed_prs: 30, per_user_reviews: [1, 2, 0]))
+        .not_to eq(0)
```

Comment:
Creo que no es suficiente ver solo que no sea 0, pondría un ejemplo con un valor numérico exacto

Reply:
Ah lo otro, para ejemplos cortos que no necesitan mayor explicación conviene poner el it inline, tipo `it { expect... }`. Así el código queda más compacto y el mensaje que entrega rspec es igual explicativo

Reply:
> Creo que no es suficiente ver solo que no sea 0, pondría un ejemplo con un valor numérico exacto

Dado que la heurística que calcula el puntaje puede cambiar, para verificar que el método sea correcto me basta que no sea cero, porque _al menos_ hace la diferencia con los casos en los que sí o sí, independiente de la heurística, tiene que dar 0 (i.e, cuando no hay otros usuarios o cuando el usuario no hace contribuciones).

Si algún día cambia la heurística, sigue estando correcto el cálculo de puntaje siempre y cuando no me de 0 cuando el usuario hace algún tipo de contribución. Dime qué opinas.